### PR TITLE
Config Workflow Diagram in adoc. Issue #210

### PR DESCRIPTION
**Pull request by joefol**

### CODE REVIEW

**Review by ruthvikm — APPROVED** [review]
Looks Good

**Review by SwindleA — APPROVED** [review]
Looks good. Im having unrelated gradle issues, so I did not test the build.

**Review by adelgadoj3 — APPROVED** [review]
Looks great!

**Review by ruthvikm — CHANGES_REQUESTED** [review]
Add a sentence that states if users want to use multiple components then the configuration should be defined in snake yaml structure

File: pi4micronaut-utils/src/docs/asciidoc/components/commun_WithHardware.adoc [hunk 1]
```diff
@@ -1,6 +1,34 @@
...
+https://github.com/oss-slu/Pi4Micronaut/edit/develop/pi4micronaut-utils/src/docs/asciidoc/components/commun_WithHardware.adoc[Improve this doc]
+
+Interacting with a hardware component is done through its communication type which determines
+its circuit setup and configuration.
+
+For example, the LCD1602 component uses an I2C communication type
+which determines how the circuit is created by utilizing the I2C pins of the GPIO.
+
+Based on the pins and communication type, you have to define the configuration in the yml file accordingly.
+If a user wants to implement multiple hardware components then the configuration should be defined in the snake yml structure.
```
Comment (ruthvikm) [inline pi4micronaut-utils/src/docs/asciidoc/components/commun_WithHardware.adoc:22]: *an user

**Review by ruthvikm — CHANGES_REQUESTED** [review]

File: pi4micronaut-utils/src/docs/asciidoc/components/commun_WithHardware.adoc (hunk 1 above)
Comment (joefol) [reply pi4micronaut-utils/src/docs/asciidoc/components/commun_WithHardware.adoc:22]: 'A user' is grammatically correct I believe

**Review by ruthvikm — APPROVED** [review]

**Pull request merged**